spec/models/miq_schedule_spec.rb

```diff
@@ -77,6 +101,17 @@
         expect(miq_schedule_array['MiqSearchContent']).not_to be(nil)
         expect(miq_schedule_array['resource_type']).to eq("MiqReport")
       end
+
+      it "imports schedule" do
+        miq_schedule_array = MiqSchedule.export_to_array([miq_schedule.id], MiqSchedule).first["MiqSchedule"]
+        imported_schedule = MiqSchedule.import_from_hash(miq_schedule_array).first
+
+        expect(imported_schedule.sched_action[:method]).to eq("run_report")
+        expect(imported_schedule.userid).to eq(user.userid)
+        expect(imported_schedule.miq_search_id).to eq(miq_search.id)
+        expect(imported_schedule.resource_type).to eq("MiqReport")
```

Comment:
for MiqSearch we can have for example "Vm".
you check it in test DB how schedule with MiqSearch  can looks like.

Reply:
with `MiqSearch`:
```
#<MiqSchedule:0x00007f865663b4b8> {
               :id => 164,
             :name => "test",
      :description => "teste",
     :sched_action => {
        :method => "vm_scan"
    },
           :filter => nil,
    :resource_type => "Vm",
           :run_at => {
                :tz => "UTC",
        :start_time => 2019-08-28 00:00:00 UTC,
          :interval => {
             :unit => "once",
            :value => ""
        }
    },
          :enabled => true,
           :userid => "admin",
     :prod_default => nil,
      :last_run_on => nil,
       :created_on => Wed, 28 Aug 2019 08:21:09 UTC +00:00,
       :updated_at => Wed, 28 Aug 2019 08:21:09 UTC +00:00,
    :miq_search_id => 11,
          :zone_id => 1,
            :adhoc => nil,
    :file_depot_id => nil,
      :resource_id => nil
}

```